test/cutting/cut_finding/test_disjoint_subcircuits_state.py

```diff
@@ -227,8 +226,10 @@ def test_CutBothWires(testCircuit):
 
     assert next_state.getSearchLevel() == 1
 
-    assert next_state.lowerBoundGamma() == 9  # 3^n scaling.
+    assert (
+        next_state.lowerBoundGamma() == 9
```

Comment:
@caleb-johnson I have kept around the bell-pair functionality for wire cuts which is what gives the 3^n scaling described here in `lowerBoundGamma()`. This was there for gate cuts as well but I noticed that you took it out recently. My feeling is that it is okay (good?) to keep this around, even though currently we do not use it to calculate our overall `gamma` since we only support LO cuts. What are your thoughts?